### DISCUSSION TITLE
Is there a way to have linux kernel coding style block comments?

### DISCUSSION THREAD

**Discussion by wesleyjrz** [numToStr/Comment.nvim#256] [Q&A] [answered]
I would like to have block comments like this:

```c
/*
 * print longest line - v1
 * read the input and print the longest line.
 */
```

However I didn't find a way to do that, I've tried to do something with ```Comment.ft``` but nothing has worked.

**Comment by numToStr** [comment, marked as answer]
Not exactly, but you can use `require'Comment.api'.insert.blockwise.eol()` which will use the block comment and if you press enter then nvim make sure to insert `*` before every new line.

**Reply by wesleyjrz** [reply]
Okay, I will try that. Thank you!

**Reply by wesleyjrz** [reply]
Will you implement a better way to have those kind of block comments?

**Reply by numToStr** [reply]
Nope. This is a specific use case that I don't want to maintain. That being said, you can make a custom comment function for your use case.
